Modix.Services/Moderation/AttachmentBlacklistBehavior.cs

```diff
@@ -1,126 +1,172 @@
+
 using Discord;
 using Discord.WebSocket;
-using Serilog;
+
+using Modix.Common.Messaging;
+using Modix.Services.Core;
+using Modix.Data.Models.Core;
 
 namespace Modix.Services.Moderation
 {
-    public class AttachmentBlacklistBehavior : BehaviorBase
+    [ServiceBinding(ServiceLifetime.Scoped)]
+    public class AttachmentBlacklistBehavior
+        : INotificationHandler<MessageReceivedNotification>
     {
         public static readonly IReadOnlyCollection<string> BlacklistedExtensions = new[]
         {
-            ".exe",
-            ".dll",
-            ".application",
-            ".msc",
-            ".bat",
-            ".pdb",
-            ".sh",
-            ".com",
-            ".scr",
-            ".msi",
-            ".cmd",
-            ".vbs",
-            ".js",
-            ".reg",
-            ".pif",
-            ".msp",
-            ".hta",
-            ".cpl",
-            ".jar",
-            ".vbe",
-            ".ws",
-            ".wsf",
-            ".wsc",
-            ".wsh",
-            ".ps1",
-            ".ps1xml",
-            ".ps2",
-            ".ps2xml",
-            ".psc1",
-            ".pasc2",
-            ".msh",
-            ".msh1",
-            ".msh2",
-            ".mshxml",
-            ".msh1xml",
-            ".msh2xml",
-            ".scf",
-            ".lnk",
-            ".inf",
-            ".doc",
-            ".xls",
-            ".ppt",
-            ".docm",
-            ".dotm",
-            ".xlsm",
-            ".xltm",
-            ".xlam",
-            ".pptm",
-            ".potm",
-            ".ppam",
-            ".ppsm",
-            ".sldn"
+                ".exe",
+                ".dll",
+                ".application",
+                ".msc",
+                ".bat",
+                ".pdb",
+                ".sh",
+                ".com",
+                ".scr",
+                ".msi",
+                ".cmd",
+                ".vbs",
+                ".js",
+                ".reg",
+                ".pif",
+                ".msp",
+                ".hta",
+                ".cpl",
+                ".jar",
+                ".vbe",
+                ".ws",
+                ".wsf",
+                ".wsc",
+                ".wsh",
+                ".ps1",
+                ".ps1xml",
+                ".ps2",
+                ".ps2xml",
+                ".psc1",
+                ".pasc2",
+                ".msh",
+                ".msh1",
+                ".msh2",
+                ".mshxml",
+                ".msh1xml",
+                ".msh2xml",
+                ".scf",
+                ".lnk",
+                ".inf",
+                ".doc",
+                ".xls",
+                ".ppt",
+                ".docm",
+                ".dotm",
+                ".xlsm",
+                ".xltm",
+                ".xlam",
+                ".pptm",
+                ".potm",
+                ".ppam",
+                ".ppsm",
+                ".sldn"
```

Comment:
Is this list the same, just indented an extra time for some reason?

Reply:
Yeah, I must have "ignore whitespace changes" enabled in VS, or I definitely would have reverted that.